src/ui/panels/clan.js

```diff
@@ -87,10 +87,14 @@ export class Clan {
         // Reset dps counter before refreshing autoclickers
         this.dashboard.currentScene.stats.resetAutoclickerDps();
         let savedClanMembers = characterData.getClanMembers();
-        for (let index = 0; index < savedClanMembers.length; index++) {
-            let memberName = savedClanMembers[index];
+        for (let memberName in savedClanMembers) {
             let member = await getAutoclickerClass(memberName, this.scrollWindow);
```

Comment:
I might've mixed up my comments, this one should be a const, the other one either needs to be a `let` or you need to create a new variable

Reply:
https://github.com/EthanVieira/rs-clicker/pull/71/commits/0828d5299ae78d1c416ad0dfa5957d3c29402087